bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotBuilder.java

```diff
@@ -44,6 +44,7 @@ public class PullRequestBotBuilder {
     private Map<String, Pattern> readyComments = Map.of();
     private IssueProject issueProject = null;
     private boolean ignoreStaleReviews = false;
+    private boolean includeSimpleMerges = false;
```

Comment:
Naming is hard. Consistent naming in an established project is harder still.

`includeSimpleMerges` (or whatever it ends up being named) is supposed to work in pair with `ignoreStaleReviews`. In that pair, "include" is the opposite to "ignore": do not ignore reviews for simple merges.

That said, I'm open to better naming.

Reply:
So you said `includeSimpleMerges` means "do not ignore reviews for simple merges."?

Reply:
Yes.

Reply:
> Just the other day, we reflected over that config option when we enabled it for the mainline jdk repository as it could be interpreted both ways, but we agreed that the opposite interpretation was our instinctive assumption.

When I started this work, I was surprised to realise that `ignoreStaleReviews` is true means that outdated reviews are **not** included in the total count of reviews. Like you said, it's counterintuitive.

But I didn't propose to change it because it's an established property that some repos already use. So, like you said, if we change it, we should be prepared to change the configuration of the affected repos too.

Those names you suggested, are they both for JSON configuration and Java code? In other words, should I change code like this?

            if (repo.value().contains("useStaleReviews")) {
                botBuilder.useStaleReviews(repo.value().get("useStaleReviews").asBoolean());
            }
            if (repo.value().contains("acceptSimpleMerges")) {
                botBuilder.acceptSimpleMerges(repo.value().get("acceptSimpleMerges").asBoolean());
            }

Reply:
Yes, I would like it to be consistent.